src/models/device.ts

```diff
@@ -91,8 +86,21 @@ export * as DeviceState from '../types/device-state';
 export type { DeviceOverallStatus as OverallStatus } from '../types/device-overall-status';
 export type { SupervisorStatus } from './device.supervisor-api.partial';
 
+type DeviceVariableModels = BalenaModel[{
```

Comment:
Let me raise that even though this looks fancy, the standard pattern would either be defining the structure